packages/react-native-worklets/Common/cpp/worklets/SharedItems/Shareables.h

```diff
@@ -204,6 +205,20 @@ class ShareableHostObject : public Shareable {
   const std::shared_ptr<jsi::HostObject> hostObject_;
 };
 
+class ShareableTurboModule : public Shareable {
+ public:
+  ShareableTurboModule(
+      jsi::Runtime &rt,
+      const jsi::Object &properties,
+      const std::shared_ptr<jsi::HostObject> &__proto__);
+
+  jsi::Value toJSValue(jsi::Runtime &rt) override;
+
+ protected:
+  std::shared_ptr<ShareableHostObject> __proto__;
```

Comment:
No need for these underscores.

```suggestion
  std::shared_ptr<ShareableHostObject> proto_;
```